hal/cloud/config.yaml

```diff
@@ -233,8 +233,39 @@ write_files:
       
       echo "algo_user setup completed successfully"
     permissions: "0755"
+    
+  # IMA setup for first boot
+  - path: /cocos_init/linux_ima_init.sh
```

Comment:
please open a pr on prism as well adding this change here https://github.com/ultravioletrs/prism/blob/main/docker/providers/cloud-init/config.yml